isis/appdata/templates/findfeatures/mosaics/00_basic_proc.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
```

Comment:
There are likely several areas where these scripts could be located. I can see why some may argue this is a tutorial rather than a template. Here is my reasoning for including them in findfeatures documentation as an example and why they are in the templates directory for findfeatures:

1. Including this as an example in findfeatures provides context. A lot of effort has been put in this work and this example demonstrates how it is used to produce a small, regional mosaic comprised of nine MESSENGER MDIS images. It explains and demonstrates why many of the design decisions were made (e.g., parameterization, an issue you raised in a previous review).
2. It is a very explicit, small example using a specific dataset. The example explains, with visuals, why the MATCH image was selected and effective application of parameters to fully utilize findfeatures capabilities and features to analyze, evaluate and understand this application. It connects context where this information is provided - in the application documentation.
3. I think putting the scripts in the `isis/scripts` directory may give the false impression they are general/ready enough to use in other circumstance. They are not. For example, the photometric correction is specific to MDIS images. However, they do provide a starting point to gain experience with findfeatures. Users are encouraged to copy, modify to suite and experiment. Putting the scripts in the template directory associates/isolates it to this application as intended.
4. We intend to include, in PRs, at least three broad, comprehensive tutorials with our ISIS APEX work when it is submitted into ISIS. These tutorials will make use of findfeatures and provide a more general, comprehensive discussion/explanation of image processing techniques that can be applied using these features and capabilities.

I hope you have had a chance to see the rendered HTML documentation. I think it is a nice addition to the findfeatures documentation to explain its role in producing mosaics.

This example is provided to help users to better understand findfeatures in a broader context. It provides a baseline configuration (i.e, template) to adjust/modify parameterization that enables quick/efficient evaluations and comparisons of its behavior.

Reply:
Thanks, this helps me understand this a little better and gives context. While reviewing I was thinking that looking over this rendered may also help see broader scope. I will do that and report back.

If we plan to introduce more comprehensive tutorials would we want to see this info eventually pointed at those or would this remain along side?

Reply:
@KrisBecker Do you have an answer to this? I think it's important to establish the purpose of these script as we need to know to what extent we need to support these scripts. I think there exists a bigger discussion as to how we want to support tutorials. And by merging these script in I believe might set a precedence we do not want to set right now as to how we want to go about supporting tutorial content.  

This also, I believe, further validates the concern of big PRs. It becomes easier to let these slip through the cracks and conflates the conversation with other PR concerns. 

For now I am gonna advocate for their removal from this PR before merging and re-introducing them in a later PR once we agree on a way to manage them.

Reply:
@Kelvinrr Simply stated, this is an example. It is an example that shows how findfeatures can be used to produce an image-based network, a concept that is heavily relied upon in the control and production of larger, regional and global mosaics.

Larger scoped tutorials would certainly refer to (the scripts in) this example to help explain what an image-based control network is and describe the basics for generating them using findfeatures. However, the pipelines used to produce the mosaics cited in the example documentation implements findfeatures' creation of these networks differently.

Just like other applications and examples, this example provides a template for users to modified as needed. It also is useful to understand how to use/apply findfeatures and the effects of its parameters.

You certainly raise very good points about discussions/guidelines/policy that need to be had regarding these types of things that currently are not available or are ill-defined. Please open an issue/discussion so that your concerns can be addressed.

Per your request, I will remove the scripts in this example and scrub mention of them from the documentation.